### ISSUE TITLE
Menu Tabs in selected state(Collapsed menu state) are not having 4.5:1 color contrast ratio

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#381]
### Steps to Reproduce

1. Launch Xamarin Workbooks application.
2. Select platform to WPF and create workbook.
3. Navigate to Menu Tabs (File/Run/Tools/Help) in collapsed state and take the screenshot. Open the screenshots in Paint.
4. Verify the color contrast ratio of Menu Tabs.

### Expected Behavior
Color contrast ratio should be minimum 4.5:1 ratio
### Actual Behavior
Color contrast ratio for copy right text is 1.3:1
### User Impact
Low vision user will feel difficult to see the text.
### Screenshots
![file](https://user-images.githubusercontent.com/22956130/40762995-9a3b6790-64c0-11e8-9b68-d5506a4e5eaf.png)
![help](https://user-images.githubusercontent.com/22956130/40762997-9a65672a-64c0-11e8-8b32-957114dd245b.png)
![run](https://user-images.githubusercontent.com/22956130/40762998-9a967c0c-64c0-11e8-83d3-a602829944c8.png)
![tools](https://user-images.githubusercontent.com/22956130/40763000-9ac1521a-64c0-11e8-85db-2c7360f44429.png)

### Environment
Windows : Version 1803 (OS Build : 17134.81)
Xamarin Version : 1.4(master/b3f92f9)

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1813
